dataverse-webapp/src/main/webapp/roles-edit.xhtml

```diff
@@ -3,34 +3,47 @@
     xmlns:ui="http://java.sun.com/jsf/facelets"
     xmlns:c="http://java.sun.com/jsp/jstl/core"
     xmlns:p="http://primefaces.org/ui">
+
     <p:dialog id="rolesPermissionsDialog" header="#{bundle['dataverse.permissions.roles.header']}" widgetVar="roleForm" modal="true" styleClass="largePopUp"> 
         <p:panel id="editRolePanel" styleClass="panelLayoutBlock" widgetVar="editRole">
             <p:panel id="editRolePropertiesPanel" styleClass="panelLayoutBlock" widgetVar="roleProperties">
                 <div class="row form-horizontal">
                     <div class="col-xs-6 form-group">
                         <label class="col-sm-3 control-label" for="roleName">
                             #{bundle['dataverse.permissions.roles.name']}
+                            <span class="glyphicon glyphicon-asterisk text-danger" title="#{bundle.requiredField}"/>
                             <span class="glyphicon glyphicon-question-sign tooltip-icon"
                                   data-toggle="tooltip" data-placement="auto right" data-container="body" data-original-title="#{bundle['dataverse.permissions.roles.name.title']}"></span>
                         </label>
                         <div class="col-sm-9">
                             <p:panelGrid columns="2" styleClass="noBorders">
-                                <p:inputText id="roleName" value="#{managePermissionsPage.role.name}"/>
-                                <p:message for="roleName"/>
+                                <p:inputText id="roleName" value="#{managePermissionsPage.role.name}"
+                                             required="#{param['DO_VALIDATION']}"
+                                             requiredMessage="#{bundle['dataverse.permissions.roles.name.requiredMsg']}"
+                                >
+                                    <f:validateLength minimum="1" />
```

Comment:
Isn't this validation do exactly the same validation as `required` param in inputText?

Reply:
It does, but i've added it to be able to manipulate length of the field to validate for more than `!empty` - IMO names of length 1 shouldn't be allowed (we can talk about it with Wojtek)

Reply:
If it does nothing then in my opinion it shouldn't be there. Also most probably text that will show if this validation fails wouldn't be presented in Polish language

Reply:
After consulting with Wojtek, we are satisfied with just `!empty` check so I'll remove `validatelength' altogether.